src/main/java/spoon/reflect/declaration/CtElement.java

```diff
@@ -224,4 +224,8 @@
 	 */
 	<E extends CtElement> E setAnnotations(List<CtAnnotation<? extends Annotation>> annotation);
 
+	/**
+	 * Deletes the statement in its block.
```

Comment:
Deletes the element. For instance, delete a statement from its containing block. Warning: it may result in an incorrect AST, use at your own risk.